projects/styles/markup/tui-space.less

```diff
@@ -3,7 +3,7 @@
 .generate-space(@direction, @i: 0) when (@i =< @sizes) {
     .generate-space(@direction, @i + 1);
 
-    &_@{direction}-@{i} {
+    &&_@{direction}-@{i} {
```

Comment:
That would do the same. We need something like `&_@{direction}-@{i}&_@{direction}-@{i}`

Reply:
Oh, yes, updated
But now it looks some unsightly (

Reply:
but I'm not sure if it will work. We need to check it out
<img width="411" alt="image" src="https://github.com/user-attachments/assets/9b5e7b16-78c1-4fa1-a887-f5643492aee0">

Reply:
@waterplea checked, yes, it's working:

<img width="690" alt="image" src="https://github.com/user-attachments/assets/1983f95a-1156-4bba-b23c-a60a14a08f73">